Jenkinsfile

```diff
@@ -1,79 +1,16 @@
 import org.kohsuke.github.GitHub
 
-def envNames = ['development', 'staging', 'production', 'devpreview', 'preview']
-
-def devBranch = 'master'
-def stagingBranch = 'master'
-def prodBranch = 'master'
-
-def isReviewable = {
-  env.BRANCH_NAME != devBranch &&
-    env.BRANCH_NAME != stagingBranch &&
-    env.BRANCH_NAME != prodBranch
-}
-
-env.CONCURRENCY = 10
-
-def isDeployable = {
-  (env.BRANCH_NAME == devBranch ||
-   env.BRANCH_NAME == stagingBranch) &&
-    !env.CHANGE_TARGET &&
-    !currentBuild.nextBuild // if there's a later build on this job (branch), don't deploy
-}
-
-def shouldBail = {
-  // abort the job if we're not on deployable branch (usually master) and there's a newer build going now
-  env.BRANCH_NAME != devBranch &&
-  env.BRANCH_NAME != stagingBranch &&
-  env.BRANCH_NAME != prodBranch &&
-  !env.CHANGE_TARGET &&
-  currentBuild.nextBuild
-}
-
-def buildDetails = { vars ->
-  """
-    BUILDTYPE=${vars['buildtype']}
-    NODE_ENV=production
-    BRANCH_NAME=${env.BRANCH_NAME}
-    CHANGE_TARGET=${env.CHANGE_TARGET}
-    BUILD_ID=${env.BUILD_ID}
-    BUILD_NUMBER=${env.BUILD_NUMBER}
-    REF=${vars['ref']}
-  """.stripIndent()
-}
+def envNames = ['development', 'staging', 'production']
```

Comment:
I only configured one built type in the metalsmith config. We'll need the three.